src/ArcCommander/IniData.fs

```diff
@@ -93,7 +99,7 @@ module IniData =
             | _                         -> createDefault (); inConfigFolder
             |> Some
         with e -> 
-            printfn "ERROR: tryGetGlobalConfigPath failed with: %s" e.Message
+            log.Error($"ERROR: tryGetGlobalConfigPath failed with:\n {e.ToString()}")
```

Comment:
Here an Error is logged, but the program does not crash afterwards like in many other cases. Maybe there could be a distinction between "Error" and "Fatal Error"?